src/Text/LLVM/PP.hs

```diff
@@ -798,19 +798,20 @@ ppCallSym ty val = pp_ty <+> ppValue val
           -> ppType res
         _ -> ppType ty
 
-ppGEP :: Bool -> Type -> Typed Value -> Fmt [Typed Value]
-ppGEP ib ty ptr ixs =
-  "getelementptr" <+> inbounds
+ppGEP :: [GEPOptionalFlag] -> Type -> Typed Value -> Fmt [Typed Value]
+ppGEP gf ty ptr ixs =
+  "getelementptr"
+    <+> (if inlineIsBool
+         then (if GEP_Inbounds `elem` gf then "inbounds" else empty)
+         else ppGepFlags gf)
```

Comment:
Possibly silly question: what would go wrong if we simplified this code to just use `ppGepFlags gf` instead? It's my understanding that having `nusw` and `nuw` attributes on `getelementptr` constants is something that only happens with LLVM 19 or later, so my expectation is that such attributes wouldn't appear in the list in older LLVM versions.

Reply:
Ah, now I see why you do this: it's because you add a `GEP_NUSW` attribute for old LLVM versions when parsing `getelementptr` constants (specifically, for `CeGepCode20`) in https://github.com/GaloisInc/llvm-pretty-bc-parser/pull/316. And that's totally fair, since `GEP_Inbounds` is supposed to imply `GEP_NUSW`.

I retract this suggestion.